stylus/components/chip.styl

```diff
@@ -21,6 +24,12 @@ $round-chip
     text-align center
     justify-content center
 
+$tiny-size-chip
+    height $chip-height-tiny
+    padding 0 (($chip-height-tiny - $chip-font-size-tiny) / 2)
+    border-radius ($chip-height-tiny / 2)
```

Comment:
Wouldn't 50% do the trick as well?

Reply:
Could the padding be done via em ? So that we have different size for free just by changing the font size ?

Reply:
https://jsbin.com/pipomijiki/1/edit?html,css,output

Reply:
@y-lohse I think we already got this conversation and no, this 50-100% trick works only with circle objects.

Reply:
@ptbrowne There is. `em` has the strongest cascade functionnality of all CSS. Your way works if there is **not any** parent that change its font-size, otherwise this won't be consistent.
Not that it can't be done but you have to be extra careful, specially when this is a component that can be called anywhere, and I don't think the economy of a few lines is worth the trouble.

Reply:
> There is. em has the strongest cascade functionnality of all CSS. Your way works if there is not any parent that change its font-size, otherwise this won't be consistent.

If we put `.Chip--normal { font-size: 1rem }`, `.Chip--xs { font-size: 0.75rem }`, it would be OK right ?

Reply:
Should be yeah, but that's not just it.
The actual different sizes are not proportionnal. The ratio `height/font-size` is not the same for each size.
- tiny = 2
- small = 2.285714286
- normal = 2.5

Reply:
Maybe we should leave a TODO comment here (and/or an issue) and propose to use proportional sizes to the design team.

Reply:
I'm not comfortable telling the designers what to do just to get a few less bytes of CSS, honestly.

Reply:
OK, thanks to have checked the ratios.

Reply:
That's why I said "propose". That's just a suggestion: if they are OK then alright, otherwise :man_shrugging: